tests/integration_tests/base_api_tests.py

```diff
@@ -115,7 +115,7 @@ def test_default_missing_declaration_put_spec(self):
         self.assertEqual(rv.status_code, 200)
         response = json.loads(rv.data.decode("utf-8"))
         expected_mutation_spec = {
-            "properties": {"id": {"format": "int32", "type": "integer"}},
```

Comment:
@dpgaspar is this a side-effect of how `marshmallow-enum` was encoding the data?

Reply:
it's actually a change done on apispec: https://github.com/marshmallow-code/apispec/pull/595

breaking change on 4.0.0

Reply:
@dpgaspar does that mean you're planning on holding off merging this until the next breaking release window?

Reply:
no, no, the 4.0.0 breaking change is on apispec, it's apispec==4.0.0, major 4 here is only a coincidence with future 4.0.0 on Superset